tests/nemo_text_processing/hy/data_inverse_text_normalization/test_cases_fraction.txt

```diff
@@ -0,0 +1,21 @@
+մեկ տասնմեկերորդ~1/11
```

Comment:
Here also context problem

Reply:
I've added some examples with context. Thank you!